### ISSUE TITLE
Format system emails to include header/footer/etc

### ISSUE THREAD

**Issue by benhaynes** [RNGR/Directus#510]
Emails coming from Directus need to have global copy (rich text is fine, but not html) for a header and footer. For instance:

* Forgot password
* New message
* Installation info
* etc

**Comment by benhaynes** [comment]
The following text should be added to the bottom of any outgoing email from Directus:
```
--
This email was sent by Directus – [Project Title](link to project)
[Log in](link to login page) to manage your email preferences
```

**Comment by WellingGuzman** [comment]
This was solved by #790

There's not actually header (content), but can also be included if necessary.

**Comment by benhaynes** [comment]
Great! Thanks Welling!
